hub/executors/encoders/image/torchvision-densenet161/torchvision_densenet.yml

```
@@ -0,0 +1,7 @@
+!ImageTorchEncoder
+metas:
+  on_gpu: true
```

I'd suggest
```suggestion
I'd suggest to set it to `false` because it is not a necessary condition.
```suggestion
  on_gpu: false
```

---

Done. I only added gpu as true as both the models work better on gpu than cpu.